### PR TITLE
Fix conditions for ESRP templates

### PR DESCRIPTION
**Pull request by adityapatwardhan**
Fix condition expression.

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
More changes are required I will make them in this PR too.

**Review by TravisEz13 — APPROVED** [review]

File: EsrpSign.yml [hunk 1]
```diff
@@ -14,16 +14,28 @@ steps:
...
-    certificateId: ${{ parameters.certificateId }}
-  condition: or(eq('${{ parameters.certificateId }}', 'CP-230012'), eq('${{ parameters.certificateId }}', 'CP-231522'))
+- pwsh: |
+    Write-Verbose -Verbose "buildOutputPath = '${{ parameters.buildOutputPath }}'"
+    Write-Verbose -Verbose "signOutputPath = '${{ parameters.signOutputPath }}'"
+    Write-Verbose -Verbose "certificateId = '${{ parameters.certificateId }}'"
+    Write-Verbose -Verbose "pattern = '${{ parameters.pattern }}'"
+  displayName: Log parameters
+
+- ${{ if eq(parameters.certificateId , 'CP-230012') }}:
```
Comment (TravisEz13) [inline EsrpSign.yml:24]: This cert should also use authenticode 'CP-231522'

**Review by TravisEz13 — APPROVED** [review]

File: template-compliance/nuget-sign.yml [hunk 2]
```diff
@@ -21,12 +21,6 @@ steps:
          operationSetCode = "NuGetSign"
          toolName = "sign"
          toolVersion = "1.0"
-        },
```
Comment (adityapatwardhan) [inline template-compliance/nuget-sign.yml:24]: could not get nuget verify to work, i will solve this later

File: EsrpSign.yml (hunk 1 above)
Comment (adityapatwardhan) [reply EsrpSign.yml:24]: having them in a since if did not work for some reason.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**